manager/allocator/networkallocator/networkallocator.go

```diff
@@ -657,7 +658,12 @@ func (na *NetworkAllocator) resolveDriver(n *api.Network) (driverapi.Driver, str
 }
 
 func (na *NetworkAllocator) loadDriver(name string) error {
-	_, err := plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	var err error
+	if pg := na.drvRegistry.GetPluginGetter(); pg != nil {
+		_, err = pg.Get(name, driverapi.NetworkPluginEndpointType, plugingetter.Lookup)
+	} else {
+		_, err = plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	}
```

Comment:
Yes. that is the idea. But plugingetter actually does its work by querying the plugins package internally if it cannot find a corresponding managed plugin. I can actually remove the call to plugin-v1 package.